<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettings</class>
 <widget class="QWidget" name="GeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>494</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QGroupBox" name="autoStartGroupBox">
     <property name="title">
      <string>Startup</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="startWengoPhoneCheckBox">
        <property name="text">
         <string>Automatically start VoxOx</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="startShowRecommendedUpdatesCheckBox">
        <property name="text">
         <string>Show recommended updates</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Idle status</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>minutes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="awaySpinBox">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="showAwayCheckBox">
        <property name="text">
         <string>Set my status to &quot;Away&quot; when I am inactive for:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>When I double-click on a contact </string>
     </property>
     <layout class="QGridLayout">
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout">
        <item row="0" column="0">
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>41</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="callCellPhoneCheckBox">
          <property name="text">
           <string>Call mobile phone or land line if contact not online.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="startFreeCallRadioButton">
        <property name="text">
         <string>Start a call</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="startChatOnlyRadioButton">
        <property name="text">
         <string>Start a chat</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>startWengoPhoneCheckBox</tabstop>
  <tabstop>callCellPhoneCheckBox</tabstop>
  <tabstop>showAwayCheckBox</tabstop>
  <tabstop>awaySpinBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
 </resources>
 <connections/>
</ui>
